homeassistant/components/wink.py

```diff
@@ -221,3 +223,11 @@ def _device_manufacturer(self):
     def _model_name(self):
         """Return the model name."""
         return self.wink.model_name()
+
+    @property
+    def _tamper(self):
+        """Return the devices tamper status."""
+        try:
```

Comment:
It is faster in Python to check if the attribute exists, so prefer to use `hasattr(self.wink, 'tamper_detected')`